packages/api/src/command/medical/patient/create-medical-record.ts

```diff
@@ -1,10 +1,9 @@
-import { MedicalRecordDateAndFormat, ResourceTypeForConsolidation } from "@metriport/api-sdk";
+import { ResourceTypeForConsolidation } from "@metriport/api-sdk";
```

Comment:
I don't think our Command should import from the SDK. I suggest we define our type here and have a schema validating the input on the route level... the shapes just need to match.